cmd/api/src/api/registration/v2.go

```diff
@@ -113,8 +114,9 @@ func NewV2API(cfg config.Configuration, resources v2.Resources, routerInst *rout
 		// Version API
 		routerInst.GET("/api/version", v2.GetVersion).RequireAuth(),
 
-		// Swagger API
+		// API Spec
 		routerInst.PathPrefix("/api/v2/swagger", v2.SwaggerHandler()),
+		routerInst.GET("/api/v2/openapi", openapi.HttpHandler),
```

Comment:
Since this is a brand new endpoint, what do you think about making it more generic? Maybe like `/api/v2/spec` or `/api/v2/api-spec`. This should keep the URL the same if we ever decide to switch spec providers or if they change their name again lol.

Reply:
i originally did use `spec`, but I can't recall why I changed it. naming things is hard :ded: